.changeset/dull-jobs-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Transaction section in order details has been improved. Now transaction navigation and management is easier.
```

Comment:
_The list of transactions is now placed under the new section called "Transactions" when you visit order details page_